docs/quickstart.md

```diff
@@ -36,3 +36,52 @@ from cai_causal_graph import EDGE_T
 # add an undirected edge between A and C
 causal_graph.add_edge('A', 'C', edge_type=EDGE_T.UNDIRECTED_EDGE)
 ```
+
+## Time Series Causal Graphs
+
+The `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph` class extends the 
+`cai_causal_graph.causal_graph.CausalGraph` class to deal with time series representations of causal graphs. The main differences with respect to the `cai_causal_graph.causal_graph.CausalGraph` class are:
+- each node is associated with a `variable_name` and a `time_lag` attribute.
+- a new `from_adjacency_matrices` method to create a `TimeSeriesCausalGraph` from a dictionary of adjacency matrices where the keys are the time lags.
```

Comment:
make sure to properly hyper-link method names and class names

Reply:
This should be done consistently in this documentation page, will stop commenting now